apps/bot-discord/src/config.json

```diff
@@ -9,24 +9,101 @@
+        "color": 14908858
+      }
+    },
     {
       "label": "revanced_download",
       "threshold": 0.85,
       "reply": {
         "title": "How to download ReVanced?",
-        "description": "You don't.",
+        "description": "ReVanced is a patcher. You can use the [ReVanced Manager](https://github.com/revanced/revanced-manager/releases/latest) or the [ReVanced CLI](https://github.com/revanced/revanced-cli) to patch the app you want to modify.\n  * [ReVanced Manager docs](https://github.com/revanced/revanced-manager/tree/main/docs#-revanced-manager)\n  * [ReVanced CLI docshttps://github.com/revanced/revanced-cli/tree/main#-documentation-and-guides-of-revanced-clihttps://github.com/revanced/revanced-cli/tree/main#-documentation-and-guides-of-revanced-cli](https://github.com/revanced/revanced-cli/tree/main#-documentation-and-guides-of-revanced-cli)",
         "color": 14908858
       }
+    },
+    {
+      "label": "androidtv_support",
+      "threshold": 0.85,
+      "reply": {
+        "title": "Will YT ReVanced support Android TVs?",
+        "description": "YouTube ReVanced for Android/Google TVs won't be made. There are alternatives, such as [SmartTubeNext](https://github.com/yuliskov/SmartTubeNext#smarttube).",
+        "color": 14908858
+      }
+    },
+    {
+      "label": "revanced_nodownloader",
+      "threshold": 0.85,
+      "reply": {
+        "title": "Why can't I download videos?",
+        "description": "You need to install a downloader to download videos. NewPipe is the default downloader, and can downloaded from [here](https://github.com/TeamNewPipe/NewPipe/releases/latest).",
+        "color": 14908858
+      }
+    },
+    {
+      "label": "revanced_casting",
+      "threshold": 0.85,
+      "reply": {
+        "title": "Why can't I cast?",
+        "description": "Because of devices like Chromecast, Android/Google TVs using the Cast v2 API, casting to those devices won't work.",
+        "color": 14908858
+      }
+    },
+    {
+      "label": "microg_download",
+      "threshold": 0.85,
+      "reply": {
+        "title": "From where can I download Vanced MicroG?",
+        "description": "You can download Vanced MicroG from [here](https://github.com/TeamVanced/VancedMicroG/releases/latest).",
+        "color": 14908858
+      }
+    },
+    {
+      "label": "microg_nointernet",
+      "threshold": 0.85,
+      "reply": {
+        "title": "Why does YT ReVanced say I'm offline?",
+        "description": "If your YT ReVanced appears to not have internet and you recently changed your Google password, remove the Vanced MicroG account from within your device settings and then log back in.",
+        "color": 14908858
+      }
+    },
+    {
+      "label": "rvdownload_unofficial",
+      "threshold": 0.85,
+      "reply": {
+        "title": "Is revanced.io official?",
```

Comment:
revanced.io is not the only unofficial site. The bot has been trained to recognize other URLs. Change it to something like "What are the official sites?". Also do this change to TG too.